src/main/java/com/aws/greengrass/lifecyclemanager/KernelLifecycle.java

```diff
@@ -54,17 +67,31 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
+import static com.aws.greengrass.componentmanager.KernelConfigResolver.CONFIGURATION_CONFIG_KEY;
+import static com.aws.greengrass.lifecyclemanager.GreengrassService.SERVICES_NAMESPACE_TOPIC;
 import static com.aws.greengrass.util.Utils.close;
 import static com.aws.greengrass.util.Utils.deepToString;
 
 public class KernelLifecycle {
     private static final Logger logger = LogManager.getLogger(KernelLifecycle.class);
-    private static final int EXECUTOR_SHUTDOWN_TIMEOUT_SECONDS = 30;
+    private static final int EXECUTOR_SHUTDOWN_TIMEOUT_SECONDS = 40;
```

Comment:
why? 30s is already quite a lot when we've interrupted the threads.

Reply:
I need to revert this default value to 30. I needed 40 for my tests.